apps/linux/tmux.talon

```diff
@@ -0,0 +1,55 @@
+os: linux
+tag: tmux
+-
+mux: "tmux "
+
+#session management
+mux new session:
+    insert('tmux new ')
+mux sessions:
+    key(ctrl-b)
+    key(s)
```

Comment:
This is just an FYI as I'll be merging this as-is, but you should be able to combine some of these into one command, unless you need arbitrary delay between
e.g.,
`key(ctrl-b s)`
`key("ctrl-b {number}")`